stream/views/management.py

```diff
@@ -32,33 +30,75 @@ def get_success_url(self):
 
 
 class RemoveView(generic.View):
-    def get(self, request, *args, **kwargs):
+    def post(self, request, *args, **kwargs):
         if not request.is_ajax():
             raise Http404
-        if request.GET['model'] == "stream":
-            stream = models.Stream.objects.get(pk=request.GET['pk'])
+        if request.POST['model'] == "stream":
+            stream = models.Stream.objects.get(pk=request.POST['pk'])
             value = False
```

Comment:
@JoeBrotzer is it possible to move the declaration of `value` outside of the `if` statement? since it is used in every condition.